scripts/bash/enceladus_env.template.sh

```diff
@@ -30,8 +30,8 @@ SPARK_SUBMIT="$SPARK_HOME/bin/spark-submit"
 
 HDP_VERSION="2.7.3"
 
-STD_CLASS="za.co.absa.enceladus.standardization.StandardizationJob"
-STD_JAR="enceladus-standardization.jar"
+STD_CLASS="za.co.absa.enceladus.conformance.StandardizationJob"
+STD_JAR="enceladus-conformance.jar"
```

Comment:
Maybe we can just merge the two `jar` variables and cut down the repetition.

Reply:
Yeah, sure. I was just thinking it can be the next step.